ironfish/src/consensus/verifier.test.ts

```diff
@@ -373,6 +373,166 @@ describe('Verifier', () => {
     })
   })
 
+  describe('Before postive block mine time activation', () => {
+    const nodeTest = createNodeTest()
+    let block: Block
+    let header: BlockHeader
+    let prevHeader: BlockHeader
+
+    beforeAll(async () => {
+      const { chain } = await nodeTest.createSetup()
+      block = await useMinerBlockFixture(
+        chain,
+        chain.consensus.parameters.disallowNegativeBlockMineTime - 1,
+      )
+      header = block.header
+      const previousBlock = block.header.previousBlockHash
+      const previousHeader = await chain.getHeader(previousBlock)
+      Assert.isNotNull(previousHeader)
+      prevHeader = previousHeader
+    })
+
+    it('fails validation when timestamp is too low', async () => {
+      jest
```

Comment:
Is this spy being used? I don't think it's doing anything. I'd double-check the extra spy usage in all the tests to make sure they are removed if they aren't being used

Reply:
Yes, the spy is used to control the Date.now() value in the verification logic. Its in `verifyBlock()` function, not all the unit tests triggered that call path tho. I will remove the ones that throw error before reach verifyBlock() function.